Build/SaveLoadScreen.cc

```diff
@@ -910,7 +1086,7 @@ static BOOLEAN DisplaySaveGameEntry(INT8 const entry_idx)
 	else
 	{
 		// If this is the quick save slot
-		wchar_t const* const txt = entry_idx == 0 ?
+		wchar_t const* const txt = entry_idx == (-1 + !gfDiDTab) ? // This is a beyond-ugly hack to remove the quicksave slot from the Dead is Dead Tab
```

Comment:
Is the const that important? I'd just set it to the empty slot and check the idx if not in DiD mode (in a separate statement).

Or perhaps create a struct for all game mode data, so also the gzGIOScreenText tests can be removed.